alacritty/src/event.rs

```diff
@@ -803,6 +807,9 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
             self.cancel_search();
         }
 
+        // We don't need IME in Vi mode.
```

Comment:
```suggestion
        // We don't want IME in Vi mode.
```

Not just that it's unnecessary, but that it would actively be harmful, right?